components/switch/index.css

```diff
@@ -10,38 +10,70 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/scales/spectrum-large.css";
-@import "../vars/css/scales/spectrum-medium.css";
-@import "../vars/css/components/spectrum-switch.css";
 
-.spectrum-Switch {
-  @remapvars {
-    find: /--spectrum-switch-m(.*)/;
-    filter: color;
-    replace: --spectrum-switch$1;
-  }
 
-  /* Hardcoded for wrapping study.
-  Will be a DNA token in https://jira.corp.adobe.com/browse/SDS-4466 */
-  --spectrum-switch-label-margin-top: var(--spectrum-global-dimension-size-65);
+.spectrum-Switch-sizeS {
```

Comment:
Thanks for adding the t-shirt sizes to this component!

Can you adjust the classes so that they're similar to the other components that have t-shirt sizes? Just need to add another dash `-` before the size modifier in each of the size classes.

```suggestion
.spectrum-Switch--sizeS {
```

Reply:
<img width="696" alt="Screen Shot 2022-08-17 at 13 49 52" src="https://user-images.githubusercontent.com/52184321/185229738-679d8b11-e2ab-46a1-8424-02c017a21898.png">